plugins/graphql/src/service/router.ts

```diff
@@ -29,17 +33,41 @@ const schemaPath = path.resolve(
 
 export interface RouterOptions {
   logger: Logger;
+  config: Config;
 }
 
 export async function createRouter(
   options: RouterOptions,
 ): Promise<express.Router> {
   const typeDefs = await fs.promises.readFile(schemaPath, 'utf-8');
 
-  const server = new ApolloServer({ typeDefs, logger: options.logger });
+  const catalogModule = await createCatalogModule(options);
+
+  const { schema } = new GraphQLModule({
+    imports: [catalogModule],
+    typeDefs,
+  });
+
+  const server = new ApolloServer({
+    schema,
+    logger: options.logger,
+    introspection: true,
+    playground: process.env.NODE_ENV === 'development',
+  });
+
   const router = Router();
 
   const apolloMiddlware = server.getMiddleware({ path: '/' });
+
+  if (process.env.NODE_ENV === 'development')
+    router.use(
+      helmet.contentSecurityPolicy({
```

Comment:
why is this needed for the graphql backend?

Reply:
This is needed because of how the playground works. I'm wondering however if we should just include use the GraphQL plugin in Backstage frontend. but then it becomes required to run the frontend and the backend at the same time.

Reply:
It's only applied on the `GraphQL` route I believe also